Parser/parser.c

```diff
@@ -18550,8 +18550,8 @@ _loop0_1_rule(Parser *p)
         D(p->level--);
         return NULL;
     }
-    ssize_t _children_capacity = 1;
```

Comment:
This file is automatically generated, you need to change the generator, not this file directly

Reply:
In the changes there is already change in generator and these changes in `parser.c` were created by `make regen-all`. Should I remove these changes to `parser.c` ? I think CI is failing without them.

Reply:
Oh, apologies, I had missed the fact they you indeed changes the generator 😅 sorry! Everything looks great 🙂